SparkPost/matchbox
# UX-437 Remove Storybook, Add Libby

This branch is deployed here: https://matchbox-libby.netlify.app/

### What Changed
- Removes Storybook
- Adds Libby, https://github.com/SparkPost/libby-react
- Most stories were unaffected, but had to change the image asset stories 
- Same with Cypress tests, had to update a couple click events

Follow up: Will need to update Netfliy CI/CD and the site with links to storybook

### How To Test or Verify
- Verify it works locally: `npm run start:libby`
- Verify Cypress tests and unit tests all pass

### PR Checklist
